### ISSUE TITLE
Make schemas shareable

### ISSUE THREAD

**Issue by jeromatron** [phact/CASTableSizer#21]
It would be nice to be able to share schemas with others, including perhaps the distributions in the second tab.  The schema itself would be fine as a start though.

**Comment by phact** [comment]
If you export the cassandra-stress.yaml you essentially have all the information about the schema and the field distributions. Does that satisfy this requirement? @jeromatron ?

**Comment by jeromatron** [comment]
Yes for sharing I think that's fine.  Also would there be value in being able to import them into the tool as well to tweak things with the interface?

**Comment by phact** [comment]
Yeah let's re-focus this issue to address import export. I'll have to add a js yaml parser but it could work nicely.
